### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -3,37 +3,49 @@
...
-//!   - light-wallets
+//!   - lightclients
 //!   - indexers
 //!   - validators
 //!
-//! as it provides a simple and configurable interface for launching and managing other processes in the
-//! local network to simulate a Zcash environment.
 //!
-//! ## List of Processes
+//! ## List of Managed Processes
```
Comment (ala-mode) [inline src/lib.rs:14]: what about `zcash-cli` and `zingo-cli`?

File: src/lib.rs [hunk 2]
```diff
@@ -3,37 +3,49 @@
...
-//! $PATH or the path to the binaries can be specified when launching a process.
+//! Ensure that any binaries manged by this crate are installed on your system.
+//! The binaries can be referenced via $PATH or the path to the binaries can be specified when launching a process.
 //!
 //! ## Testing
 //!
 //! Pre-requisities for running integration tests successfully:
 //! - Build the Zcashd, Zebrad, Zainod and Lightwalletd binaries and add to $PATH.
-//! - In order to geneate a cached blockchain from zebrad run:
+//! - In order to generate a cached blockchain from zebrad run:
```
Comment (ala-mode) [inline src/lib.rs:29]: maybe add something like,
"... and see the resulting change in local state ..."

File: src/lib.rs [hunk 3]
```diff
@@ -3,37 +3,49 @@
...
 //! ```BASH
-//! cargo nextest run generate_zebrad_large_chain_cache --run-ignored ignored-only --features test_fixtures
+//! ./utils/compare_chain_caches.sh
 //! ```
+//! This command generates new data in the `chain_cache` directory.  The new structure should have the following added
```
Comment (ala-mode) [inline src/lib.rs:33]: or put that same type of info here, to explain what the `.sh` does

I think that would be more clear, even though you are talking about 'new structure' here.

File: utils/launch_testnet_zebrad.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+set -e -x
+rm -rf testnet_zebrad.toml
+zebrad generate --output-file testnet_zebrad.toml 
+sed -i 's/Mainnet/Testnet/g' testnet_zebrad.toml
+sed -i 's/listen_addr = "0.0.0.0:8233"/listen_addr = "0.0.0.0:18233"/g' testnet_zebrad.toml
+
+zebrad --config testnet_zebrad.toml start
```
Comment (ala-mode) [inline utils/launch_testnet_zebrad.sh:7]: I'll check out this branch and work with this script and follow up soon.

**Review by ala-mode — CHANGES_REQUESTED** [review]
Looks over all good, a few minor changes requested and I'm going to check out that new util script.

File: utils/launch_testnet_zebrad.sh [hunk 5]
```diff
@@ -0,0 +1,7 @@
+set -e -x
+rm -rf testnet_zebrad.toml
```
Comment (ala-mode) [inline utils/launch_testnet_zebrad.sh:2]: why `rm -rf` here?
could we just use `rm ./testnet_zebrad.toml` ?

File: src/lib.rs (hunk 1 above)
Comment (zancas) [reply src/lib.rs:14]: Maybe add them to the list later..  I'm not sure how they're used-or-not.

File: src/lib.rs (hunk 3 above)
Comment (zancas) [reply src/lib.rs:33]: Will likely take this suggestion in a subsequent PR, though the doc system doesn't render utility script comments nicely.

File: utils/launch_testnet_zebrad.sh (hunk 5 above)
Comment (zancas) [reply utils/launch_testnet_zebrad.sh:2]: `-f` succeeds even if there's nothing there.   The `r` isn't doing anything.

**Review by ala-mode — APPROVED** [review]

**Comment by ala-mode** [comment]
Thanks for looking at my feedback. Merging.

**Pull request merged**